src/index.css

```diff
@@ -0,0 +1,113 @@
+/* General style */
```

Comment:
While I can see the effort you've made, your project does not look like the given design yet.

Your Implementation | Zeplin Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/155289389-252b2042-0181-4bf4-b293-088b9eab5a6e.png) | ![image](https://user-images.githubusercontent.com/57421823/155289520-03906370-3964-40b5-93a5-7a4327641efb.png)

Therefore:

- [ ] Please, as requested by the previous reviewer, mind the font styles, margins, spacings, padding given in the Zeplin design.  Your design should match the Zeplin design very closely.

This is how you do can easily do it:

![image](https://user-images.githubusercontent.com/57421823/155289106-0260b188-adc0-471c-aecb-f4654c4cf961.png)

On the Zeplin, select each element, then look to your right and almost all the styles will be there. 💪